inscriptions/ISic000593.xml

```diff
@@ -185,11 +190,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/><name><expan><abbr>D</abbr></expan></name> <g ref="#interpunct">·</g> <name>Sempron</name><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
first name is Decimus also?
Sempron needs a gap immediately following wihtin the name, and then a further gap afterwards

Reply:
I think my problem was that I was unsure whether it would be in the nominative case here. I'll render it as a cert="low" abbreviation, unless you think its pretty likely?